### PR TITLE
BWS-76: + NotFound page

### PR DESCRIPTION
**Pull request by bohdanshcherba**

### CODE REVIEW

File: frontend/src/components/not-found-page/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+.link {
+  color: var(--color-orange);
+  text-decoration: none;
+}
+
+.link:active {
+  color: var(--color-orange);
+}
```
Comment (what1s1ove) [inline frontend/src/components/not-found-page/styles.module.scss]:
```suggestion
```
we should have orange by default

File: frontend/src/components/not-found-page/not-found-page.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { FC } from 'react';
+import styles from './styles.module.scss';
+import { AppRoute } from '../../common/enums/app/app-route.enum';
+import { Link } from '../common/link/link';
+
+const NotFound: FC = () => {
+  return (
+    <div className={styles.wrapper}>
+      <div>
+        <h5 className={styles.title}>Page Not Found</h5>
```
Comment (what1s1ove) [inline frontend/src/components/not-found-page/not-found-page.tsx]:
```suggestion
        <h1 className={styles.title}>Page Not Found</h1>
```

File: frontend/src/components/not-found-page/not-found-page.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { FC } from 'react';
+import styles from './styles.module.scss';
+import { AppRoute } from '../../common/enums/app/app-route.enum';
+import { Link } from '../common/link/link';
+
+const NotFound: FC = () => {
+  return (
+    <div className={styles.wrapper}>
+      <div>
```
Comment (what1s1ove) [inline frontend/src/components/not-found-page/not-found-page.tsx]:
```suggestion
```
avoid useless wrappers

File: frontend/src/components/not-found-page/not-found-page.tsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import { FC } from 'react';
+import styles from './styles.module.scss';
+import { AppRoute } from '../../common/enums/app/app-route.enum';
```
Comment (what1s1ove) [inline frontend/src/components/not-found-page/not-found-page.tsx]:
```suggestion
import { AppRoute } from '../../common/enums/enums;
```

File: frontend/src/components/not-found-page/not-found-page.tsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+import { Link } from '../common/link/link';
```
Comment (what1s1ove) [inline frontend/src/components/not-found-page/not-found-page.tsx]:
```suggestion
import { Link } from 'components/common/common';
```

File: frontend/src/components/app/app.tsx [hunk 6]
```diff
@@ -45,6 +46,7 @@ const App: FC = () => {
   return (
     <>
       <Routes>
+        <Route path={AppRoute.ALL} element={<NotFound />} />
```
Comment (what1s1ove) [inline frontend/src/components/app/app.tsx]:
```suggestion
        <Route path={AppRoute.ALL} element={<AuthorizedRoute component={<NotFound />} />
        
```

Comment (what1s1ove) [inline frontend/src/components/app/app.tsx]: should be at the bottom of all routes

File: frontend/src/common/enums/app/app-route.enum.ts [hunk 7]
```diff
@@ -8,6 +8,7 @@ enum AppRoute {
   BS = '/bs',
   SC = '/sc',
   SLC = '/slc',
+  ALL = '*',
```
Comment (Yaroslaww-1) [inline frontend/src/common/enums/app/app-route.enum.ts]: NOT_FOUND = '*',

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: frontend/src/components/not-found-page/not-found-page.tsx [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+
+const NotFound: FC = () => {
```
Comment (Yaroslaww-1) [inline frontend/src/components/not-found-page/not-found-page.tsx:6]: Please add page screenshot to PR

**Review by Yaroslaww-1 — CHANGES_REQUESTED** [review]

**Comment by bohdanshcherba** [comment]
image

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**